output/Form.md

```diff
@@ -23,10 +24,39 @@ title: Yerun Open Science Award form
 -   [*Benefits*]{.underline}*: What benefits were realised (to date) through
     this initiative, and for whom?*
 
+As a leading international research group and a World Health Organization's
+collaborating centre, we work daily with researchers and professionals living
+abroad. This requires tools that help tracking exchanges between labs in a
+comprehensive way, such as statistical code, manuscript drafts, or meeting
+minutes. By taking this approach, we have not only foster a stronger open
+science practice within our lab, but also across different research consortia in
+the European Union (EU). This may potentially spread through different
```

Comment:
This sounds great and is very well written. However, I have the concern that this is not true, at least up to now, Remember we are talking here about "what we have achieved so far".